apps/OboeTester/app/src/main/cpp/analyzer/LatencyAnalyzer.h

```diff
@@ -73,8 +73,8 @@ static double calculateNormalizedCorrelation(const float *a,
 
     // Correlate a against b.
     for (int i = 0; i < windowSize; i++) {
-        float s1 = a[i];
-        float s2 = b[i];
+        double s1 = a[i];
```

Comment:
This code is already too slow.
Maybe better performance to make correlation, sumProducts, and sumSquares all float.